notebooks/samples/Regression - Flight Delays.ipynb

```diff
@@ -102,8 +114,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import random\n",
-    "model_name = \"dbfs:/flightDelayModel.mml\"\n",
+    "if os.environ.get(\"AZURE_SERVICE\", None) == \"Microsoft.ProjectArcadia\":\n",
+    "    model_name = \"abfss://gatedbuild@mmlsparkgatedbuild.dfs.core.windows.net/models/flightDelayModel.mml\"\n",
```

Comment:
Can we make this some place that makes sense for a random user? Like a local path?

Reply:
solved